changelog/7584.feature.md

```diff
@@ -0,0 +1 @@
+Adds the capability for creating an augmented dataset. Data suggestions are based on paraphrases and chosen on the basis of low-data and/or low-performing intents.
```

Comment:
Can we add a link to the paraphrasing repo as well here, just so that users know that this feature is to be used together with it.